src/backend/postgres/index.rs

```diff
@@ -65,4 +75,25 @@ impl IndexBuilder for PostgresQueryBuilder {
             write!(sql, "UNIQUE ").unwrap();
         }
     }
+
+    fn prepare_table_ref(&self, table_ref: &TableRef, sql: &mut SqlWriter) {
+        match table_ref {
+            TableRef::Table(table) => {
+                table.prepare(sql, self.quote());
+            }
+            TableRef::SchemaTable(schema, table) => {
+                schema.prepare(sql, self.quote());
+                write!(sql, ".").unwrap();
+                table.prepare(sql, self.quote());
+            }
+            TableRef::DatabaseSchemaTable(database, schema, table) => {
+                database.prepare(sql, self.quote());
+                write!(sql, ".").unwrap();
+                schema.prepare(sql, self.quote());
+                write!(sql, ".").unwrap();
+                table.prepare(sql, self.quote());
+            }
```

Comment:
I don't think we can create index across database? i.e. 
```sql
CREATE INDEX idx_aaa 
ON aaa.public.accounts (password); # `public.accounts` is valid but not `aaa.public.accounts`
```
_https://dbfiddle.uk/?rdbms=postgres_14&fiddle=67b77e58c1c524174d1675ff92511e2d_

Reply:
That was removed :) Thanks for catching this. I think I probably missed testing this in my test DB 🙏